### DISCUSSION TITLE
Join GitHub Education

### DISCUSSION THREAD

**Discussion by Kentaki1607** [community/community#60877] [GitHub Education]
### Hi everyone

I have a question about joining GitHub Education. My problem is that my university does not provide student email accounts. So I don't have a school-issued email address to enter into the registration page. Is there any other way to verify that I'm a student of that university to use GitHub Education? Thanks

**Comment by nonvenomous** [comment]
Same issue here, [Required mail from institution but institution does not provide it #60650](https://github.com/orgs/community/discussions/60650).
No known solution so far.

**Comment by the-officialjosh** [comment]
There is no other sure way to get it done, although it is possible to get around to submitting but getting approved it highly unlikely. While I was in College I had to write to my IT department and even my Head of Department for it and was granted a temporary one for this case. You could try writing and make it private so as not to get your request thrown out because everyone wants to get one.

**Comment by samus-aran** [comment]
Hi @Kentaki1607 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Evidence requirement is based on the school you are applying for. If your school doesn't not distribute academic emails but our verification states otherwise then please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍

Hope this helps! Let us know how you get along 👍
